database/db.sql

```diff
@@ -5,10 +5,11 @@ CREATE TABLE users (
     firstName VARCHAR (50) NOT NULL,
     lastName VARCHAR (50) NOT NULL,
     email VARCHAR (50) UNIQUE NOT NULL,
-    password VARCHAR (50) NOT NULL,
+    password VARCHAR (255) NOT NULL,
     gender VARCHAR (50) NOT NULL,
     jobRole VARCHAR (50) NOT NULL,
     department VARCHAR (50) NOT NULL,
     address VARCHAR (50) NOT NULL,
     isAdmin BOOLEAN
```

Comment:
use camel case in this. don't just copy code do proper research

Reply:
snake case you mean sir?